src/404.html

```diff
@@ -0,0 +1,61 @@
+---
+title: "We couldn't find that Web page (Error 404) / <span lang='fr'>Nous ne pouvons trouver cette page Web (Erreur 404)</span>"
+permalink: 404.html
+---
+<!DOCTYPE html>
+<html class="no-js" lang="en" dir="ltr">
+
+	<head>
+		<meta charset="utf-8" />
+		<!-- Web Experience Toolkit (WET) / Boîte à outils de l'expérience Web (BOEW)
+		wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html -->
+		<title>{{ title | striptags(true) | escape | nl2br }}</title>
+		<meta content="width=device-width, initial-scale=1" name="viewport" />
+		<meta name="robots" content="noindex, nofollow, noarchive" />
+		<link href="https://wet-boew.github.io/themes-dist/GCWeb/GCWeb/assets/favicon.ico" rel="icon" type="image/x-icon" />
+		<link rel="stylesheet" href="https://wet-boew.github.io/themes-dist/GCWeb/GCWeb/css/theme-srv.css" />
+
+	</head>
+
+	<body vocab="http://schema.org/" resource="#wb-webpage" typeof="WebPage">
+		<header id="wb-bnr" class="container">
+			<div class="row" property="publisher" resource="#wb-publisher" typeof="GovernmentOrganization">
+				<div class="col-sm-6">
```

Comment:
To be consistent with our splash page, apply "text-right".

```suggestion
				<div class="col-sm-6 text-right">
```
ref. https://github.com/wet-boew/GCWeb/blob/master/templates/splash/splashpage-en.md?plain=1#L29